test/runtests.jl

```diff
@@ -4,7 +4,7 @@ using StaticArrays
 using Test
 using LinearAlgebra
 
-using Distances: Distances, Metric, evaluate, PeriodicEuclidean
+using Distances: Distances, Metric, MinkowskiMetric, evaluate, PeriodicEuclidean
```

Comment:
You could load `UnionMinkowskiMetric`, and define a `const MinkowskiMetric = UnionMinkowskiMetric`, right? Would that solve the problem, without changing other parts of the code? That would even keep the name `NearestNeighbors.MinkowskiMetric` available.